Core/FunctionTree/FunctionTree.cpp

```diff
@@ -2,19 +2,20 @@
 // This file is part of the ComPWA framework, check
 // https://github.com/ComPWA/ComPWA/license.txt for details.
 
-#include "Core/FunctionTree.hpp"
+#include "FunctionTree.hpp"
```

Comment:
Shouldn't that be `Core/FunctionTree/FunctionTree.hpp`?

Reply:
I just removed the full path of all includes from the same module, since that makes it a bit easier when modules alltogether

Reply:
Didn't you also change in install location? The reason why the full path as included in the header files is that after installation the path also needs to be correct.

Reply:
I see. But inside a cpp file this would not matter. I thought the compiler will always find the header file if its in the same directory as the current header file (that includes the new header). And since these modules are always bundled together then there should be no problem (https://gcc.gnu.org/onlinedocs/cpp/Search-Path.html)

Reply:
Build ComPWA is of course not a problem. But the header files are installed in a directory structure. And within this structure the path also needs to be correct. E.g. we have two header files which are installed in `$INSTALL_DIR`:
```
$INSTALL_DIR/include/ComPWA/Core/Parameter.hpp
$INSTALL_DIR/include/ComPWA/Core/ParameterList.hpp
```
If I now include `ParameterList.hpp` in another project `Parameter.hpp` needs to be found. I add the include path `include/ComPWA`. If we place `#include "Parameter.hpp"` in `ParameterList.hpp. This will not work.

Reply:
I thought that this does work... I will try it

Reply:
Yeah this works for me. I don't understand the reasoning behind this. But I just checked the gnu include files and they also specify the full path (at least where I looked). Well I guess there wont be any confusion, in case there would be multiple files with the same name. I will just change it to the full path.